### ISSUE TITLE
Accessibility: component progress have not  sufficient color contrast

### ISSUE THREAD

**Issue by mouracamila** [themesberg/flowbite-react#178]
**Describe the bug**
In the storybook there is an "incomplete" message regarding the background 
color of the progress component, this happens in the dark an d the light mode

**To Reproduce**
1. Go to Storybook
2. Click on Progress/Sizes
3. Scroll down and click on the Accessibility option

**Expected behavior**
Elements must have sufficient color contrast

**Screenshots**
![Screenshot-02](https://user-images.githubusercontent.com/41678283/171720491-59fd8e4a-0347-46a6-8fb1-6d34a18fc320.png)
![Screenshot from 2022-06-02 21-14-40](https://user-images.githubusercontent.com/41678283/171720552-c7c3da1f-a8ff-4f41-b784-bfd3b12b1ebe.png)

**Desktop (please complete the following information):**
 - Browser Firefox

**Additional context**

**Comment by rluders** [comment]
Thank you for reporting this issue. However, we regret to inform you that we won't be able to address it for the following reasons:

- The reported issue refers to an unsupported old version of our project.
- Our team was unable to reproduce the problem with the information provided.
- Insufficient information was provided to conduct a thorough investigation.

We recommend upgrading to the latest version and submitting a new issue if the problem persists. Thank you for your understanding.
